create-integration-deployment/action.yml

```
@@ -5,8 +5,13 @@ inputs:
   integrationName:
     description: "Integration name as it appears in the integration-deployments repo"
     required: true
+  graphProjectName:
+    description: "Graph project name"
+    required: true
```

Is there any way to make this optional, so that it's not a breaking change?

---

For example, could we use the `||` operator? ([link](https://docs.github.com/en/actions/learn-github-actions/expressions#operators))

```suggestion
  graphProjectName:
    description: "Graph project name"
```

```diff
-         yarn upgrade @jupiterone/graph-${{ inputs.graphProjectName || inputs.integrationName }}@^${{ inputs.version }}
+         yarn upgrade @jupiterone/graph-${{ inputs.graphProjectName || inputs.integrationName  }}@^${{ inputs.version }}
```

---

Just tested and this would work.